server/utils/chats/index.js

```diff
@@ -104,7 +104,9 @@ async function chatWithWorkspace(workspace, message, chatMode = "chat") {
       error: null,
     };
   } else {
-    const rawHistory = await WorkspaceChats.forWorkspace(workspace.id, 20);
+    var chat_memory = workspace?.openAiHistory;
+    // console.debug(chat_memory);
```

Comment:
can we remove this commented out console log?

Reply:
Also for JS code we utilize `camelCase` naming convention. I also think a name like `messageLimit` would be more indicative of what it is doing as chat memory sound like actual memory objects.

Reply:
camelCased and removed comment